src/store/ducks/generic/plans.ts

```diff
@@ -0,0 +1,174 @@
+import { get, keyBy, values } from 'lodash';
```

Comment:
This is just an IRS plans reducer module.  It is not "generic"

Reply:
Are the new jurisdiction reducer module "generic"? Was there a particular reason behind naming the directory `generic`?

Reply:
@cKellyDesign actually none whatsoever, I had named it "IRS" at first but then that didnt seem appropriate

Reply:
@moshthepitt how about we change `generic` to `custom`? This should type of vernacular should be considered in #464.

Reply:
@cKellyDesign agreed

Reply:
Change of heart here, it seems the generic reducer/typings/methods should remain `generic`, while we shift the custom typings out of the generic code (and ideally dynamically generate them from static configs in `settings.ts`).

I don't think this needs to happen in this PR, issue created at https://github.com/onaio/reveal-frontend/issues/468